tests/unit/pages/Dashboard.spec.js

```diff
@@ -122,6 +122,14 @@ describe("Dashboard.vue", () => {
       expect(wrapper.find('[data-test="surveyQuestionWarning"]').exists()).toBe(
         global.dummyPlioMetrics["has_survey_question"]
       );
+
+      global.dummyPlioMetrics["has_survey_question"] = false;
+
+      mountWrapper();
+
+      expect(wrapper.find('[data-test="surveyQuestionWarning"]').exists()).toBe(
+        global.dummyPlioMetrics["has_survey_question"]
+      );
```

Comment:
this updates the `global.dummyPlioMetrics` value and if it is used in other test cases that run after this one (if we add any in the future), it would give incorrect results. And it is not the right way to test in the first place - you can create another dummy data object for survey questions being false and pass it in separately + there should be a different test case function written for this use case